src/lib/index.js

```diff
@@ -1145,6 +1176,14 @@ function templateFormatString(string, labels, d3locale) {
             }
         }
 
+        // Apply mult/div operation (if applicable)
+        if(parsedOp) {
+            value = (value === undefined) ? undefined : {
```

Comment:
If you test against ` value !== undefined` in the upper condition, this logic would be simplified.